backend/db.json

```diff
@@ -7,6 +7,7 @@
     "GameSequence" : [
       {
         "ArtifactName" : ["Barnacle"],
+        "NumberId" : "1",
```

Comment:
Thoughts on changing to 'ArtifactId'? Its possible that they might not want numeric IDs, and may want characters and or numbers.

Reply:
That sounds fine to me.